package.json

```diff
@@ -25,17 +34,18 @@
     "standalone:replay": "REPLAY=replay cozy-konnector-standalone src/index.js",
     "build": "webpack",
     "format": "prettier --write '**/*.{js,json}'",
-    "precommit": "pretty-quick --staged",
-    "deploy":
-      "git-directory-deploy --directory build/ --branch build --repo=https://github.com/konnectors/cozy-konnector-mgen.git",
-    "deploy:prod":
-      "git-directory-deploy --directory build/ --branch latest --repo=https://github.com/konnectors/cozy-konnector-mgen.git"
+    "precommit": "eslint . && pretty-quick --staged",
+    "lint": "eslint .",
+    "deploy": "git-directory-deploy --directory build/ --branch build --repo=https://github.com/konnectors/cozy-konnector-mgen.git",
+    "deploy:prod": "git-directory-deploy --directory build/ --branch latest --repo=https://github.com/konnectors/cozy-konnector-mgen.git"
   },
   "dependencies": {
     "cozy-konnector-libs": "3.3.0"
   },
   "devDependencies": {
     "copy-webpack-plugin": "4.4.1",
+    "eslint": "4.18.0",
+    "eslint-config-prettier": "2.9.0",
```

Comment:
Have you seen this comment ? https://github.com/konnectors/cozy-konnector-ameli/pull/32#issuecomment-366898404

It is not very important as we will be able to change every konnector in a single pass if they follow the same structure.